`azurerm_subnet` - adding property `default_outbound_access_enabled` (#25259)
hashicorp/terraform-provider-azurerm: website/docs/r/subnet.html.markdown

* `delegation` - (Optional) One or more `delegation` blocks as defined below.

> * `default_outbound_access_enabled` - (Optional) Enable default outbound access to the internet for the subnet. Setting to `false` creates a private subnet.

Reviewer: ```suggestion
* `default_outbound_access_enabled` - (Optional) Specifies where this Subnet has default outbound access to the internet. Defaults to `true`.
```
